lib/orbit_web/router.ex

```diff
@@ -15,13 +15,15 @@ defmodule OrbitWeb.Router do
 
     plug :put_secure_browser_headers, %{
       # much of this is necessary for Appcues: https://docs.appcues.com/user-experiences-faq/faq-content-security-policies
+      # script-src-elem 'unsafe-inline' is for react developer tools
```

Comment:
Can this comment go above 26, or does `mix format` dislike that?

Reply:
content security policy is one long string from line 19 to 28 (note the backslashes), so it's syntactic issue that you can't put a comment in the middle of a string, not a formatting issue.